packages/main/test/samples/Slider.sample.html

```diff
@@ -0,0 +1,48 @@
+<header>
+	<div class="control-header">Slider</div>
```

Comment:
Use the following header, so the since is also displayed

```html
<header class="component-heading">
	<h2 class="control-header">Slider</h2>
	<div class="component-heading-since">
		<span><!--since_tag_marker--></span>
	</div>
</header>
```